.github/workflows/cmake-multi-platform.yml

```diff
@@ -171,7 +234,7 @@ jobs:
         -DOPENMP_LIBRARIES="/opt/homebrew/opt/llvm/lib"
         -S ${{ github.workspace }}
 
-    - name: Build without Windows
+    - name: Build (Unix)
       # Build your program with the given configuration. Note that --config is needed because the default Windows generator is a multi-config generator (Visual Studio generator).
       if: ${{ !startsWith(matrix.os,'windows-') }}
       run: sudo cmake --build ${{ steps.strings.outputs.build-output-dir }} --config ${{ matrix.build_type }}
```

Comment:
The windows build fails with the Google Test Framework enabled, i.e., the second build. The first build without the test framework enabled works just fine. The reason is most likely a configurational issue within the CMake files that I will investigate.

<img width="1666" alt="Screenshot 2023-12-07 at 6 20 35 AM" src="https://github.com/franziska-wegner/egoa/assets/57569315/cd269206-d822-422a-a8da-554d104ee158">

Reply:
Adding the right compiler flags for Windows solved this particular issue. We did this using `CMAKE_MSVC_RUNTIME_LIBRARY`, which only enables when setting the right policy. See https://cmake.org/cmake/help/latest/variable/CMAKE_MSVC_RUNTIME_LIBRARY.html#variable:CMAKE_MSVC_RUNTIME_LIBRARY for more information.

```
cmake_policy(SET CMP0091 NEW)

# Select the MSVC runtime library
set(CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreadedDLL")
```

The only problem that still exists is that the DeathTest fails with Windows debug.